### ISSUE TITLE
"Drop with force" Button from UI that fixes "ERROR:  cannot drop the currently open database"

### ISSUE THREAD

**Issue by codingyourlife** [pgadmin-org/pgadmin4#5714]
**Describe the solution you'd like**
I want to right click drop Delete/Drop and there should be a third button "Drop with Force" that just ignores all in use connections and force drops. For me (see screenshot) not even with force works but I think the command line version works and I usually use that.

**Additional context**
I often drop and recreate a database. The problem that I run into is that tables already exist and therefore it fails. I don't care about existing tables and connections. I just want to recreate from backup to retry e.g. deleting an entity entry with changed codebase. My problem is that the database is somehow in use so I run into the error:
```
ERROR:  cannot drop the currently open database
SQL state: 55006
```

I am annoyed repeatedly by this and I didn't find an issue about it so I created this one now.

My current version: Version 6.1 (4280.88)

<img width="589" alt="Screenshot 2023-01-05 at 14 24 35" src="https://user-images.githubusercontent.com/10122382/210790221-eb1af5e8-3700-4662-9e78-5a46b41620a4.png">

**Comment by codingyourlife** [comment]
Update: I tried
brew install --cask pgadmin4
-> still old version

I then tried:
brew uninstall --cask pgadmin4
brew install --cask pgadmin4

and then I had latest Version 6.18 (4280.88) and could just delete the database without problems.

So potentially solved potentially coincidence... Above is therefore potentially a workaround. I leave open but feature can potentially be closed after some time.

**Comment by codingyourlife** [comment]
Ok not fixed second time I now run into this message.
Again I don't care I just want to recreate the db and kill active connections. This should be more convenient.
![image](https://user-images.githubusercontent.com/10122382/210795455-ba482719-a7fc-4017-bfa8-46df7efc3832.png)

**Comment by dpage** [comment]
I'm re-opening this. Giving the option to FORCE the drop does not seem unreasonable - obviously that would need to be executed via the maintenance DB connection, and could not be offered for the maintenance DB itself, because you cannot force-drop the database you're connected to.

**Comment by codingyourlife** [comment]
I'm again annoyed by this today...
seems you guys are generally in for providing a fix. thanks for that!

so this is my workaround:

recreateDb.sh:
`psql -U postgres -a -f "recreateDb.sql";`

recreateDb.sql:
```
DROP DATABASE IF EXISTS "my-db-name" WITH (FORCE);
CREATE DATABASE "my-db-name";
```

Hope this helps.

<img width="681" alt="Screenshot 2023-01-20 at 22 27 57" src="https://user-images.githubusercontent.com/10122382/213808480-3e93dd86-8430-4398-8278-e141f4976ec6.png">

**Comment by pravesh-sharma** [comment]
Hi @codingyourlife,

This is fixed with #6367.
